### PR TITLE
#42: add __call__ to Actor.

### PR DESCRIPTION
**Pull request by perrygoy**
Addressing #42.

This PR adds a `__call__` method to `Actor`, which allows you to even more succinctly give the Actor a list of things to do:

```python
the_actor(Sleeps.for_(10).seconds_because("I'm doing a demonstration."))

the_actor(
    Clicks.on_the(LOGIN_BUTTON),
    Eventually(Enters.the_text(PASSPHRASE).into_the(PASSPHRASE_FIELD)),
    Clicks.on_the(SUBMIT_BUTTON),
)

the_actor(Asserts.the(Text.of_the(ACCOUNT_BANNER), ContainsTheText("Welcome!")))
```

For this, obviously we needed a bevy of present-tense aliases for our existing Actions, and those will need to be added for the extensions as well.

### CODE REVIEW

File: screenpy/actor.py [hunk 1]
```diff
@@ -165,43 +165,43 @@ def attempts_to(self: SelfActor, *actions: Performable) -> None:
             self.perform(action)
 
     def was_able_to(self: SelfActor, *actions: Performable) -> None:
-        """Alias for :meth:`~screenpy.actor.Actor.attempts_to`, for test setup."""
+        """Alias for :meth:`~screenpy.actor.Actor.attempts_to`."""
```
Comment (perrygoy) [inline screenpy/actor.py:168]: I also thought i'd take this prescriptive "you should use it here" stuff out. Folks can use these wherever they want!

File: tests/test_actor.py [hunk 2]
```diff
@@ -33,6 +35,7 @@ def test_calls_perform_as() -> None:
     action.perform_as.assert_called_once_with(actor)
 
     perform_aliases: list[Any] = [
+        actor,
```
Comment (perrygoy) [inline tests/test_actor.py:37]: It was awesome to be able to just add this here, nice test @bandophahita!

Comment (bandophahita) [reply tests/test_actor.py:37]: it's because I'm so lazy.

**Review by bandophahita — APPROVED** [review]
This looks good!

**Pull request merged**